plugins/clipboard/plugin.js

```diff
@@ -427,12 +440,19 @@
 		 * an upcoming major release.
 		 */
 		editor.getClipboardData = function( callbackOrOptions, callback ) {
```

Comment:
Function signature once again changed, now we're using options once again…

That's a messy situation and we should adjust the API docs correctly. I mean that the options argument is again supported, etc.

Reply:
I've just dropped handling of options as it was used only to set custom title of paste dialog, which wasn't even working.